### ISSUE TITLE
Failures on some Notifications and Modules.

### ISSUE THREAD

**Issue by minnsch** [nabeelio/phpvms#1923]
**Describe the bug**
Two core bug I'm experiencing after transfering my Virtual Airlines from Subdomain to my Main Domain. Neither have we skipped or missed something I'm not quite sure. Bugs we are experiencing as below:

1. When changing ranks & awards, the Discord Webhook we setup (even when I changed the link to a new one to make sure) will not send at all. No logs record or way to debug. Error logs didn't appear either.

2. Overall Discord Webhook doesn't work: Administrative Mails sent to VA Private Webhook fails and didn't send anything since.

3. Sending a news with "Send Notifications" didn't send any mails. But Verification and Password Reset Request are working as its intended. (MailerSend is in use and it never fails before the transfer)

4. Mails Fails to send Administrative Emails: such as New Registration being made are not sent to admin mails which before it does.

**Version**
version 7.0.0-dev.ge29f1651e8 - Latest Updated before trying to reproduce myself again.

**To Reproduce**
Steps to reproduce the behavior for issue number one:
1. Make an awards and ranks. (Blank awards or other, category didn't matter)
2. Webhook setup correctly + Discord "Award Message" and "Rank Change" for Public are ticked.
3. Assign an award using Disposable Basic "Manual Award" or Directly Assigning to a User or setting a parameter value closest to my statistic to check. (Automatically assigned or Manually assigned both didn't matter)
4. Discord Webhook should be appearing but it didnt. (For this no error logs or any other logs was found)

Steps to reproduce the behavior for issue number three:
1. Writing a valid news and sending it with "Send Notifications" ticked.
2. On the phpVMS settings make sure to have "News Mails" ticked.

Steps to reproduce the behavior for issue number two and four:
1. Setup Webhook (Admin/Private) and Mailings correctly.
3. Make a new registration.

**Expected behavior**
It doesn't send any webhooks or emails.

**Additional context**
Additionally for DisposableBasic Whazzup feature, from the logs its showing that Whazzup is reading and downloading correctly. However, the widget remained showing "No VATSIM flights has found".

Logs Attached Showed Something regarding all of the Issues Explained Above -> [laravel-2024-12-30.log](https://github.com/user-attachments/files/18277806/laravel-2024-12-30.log)

**Comment by nabeelio** [comment]
Did you update the .env file for the URL? And any Discord URLs, etc?

**Comment by minnsch** [comment]
> Did you update the .env file for the URL? And any Discord URLs, etc?

Affirmative, .env file is updated.
App URL, Discord oAuth.

Discord URLs stay the same as we are using the same Webhooks, for the testing I did make another webhook and insert it back but all of the URLs are confirmed the correct one.

Moreover, my web is currently running normally other than those reported issue.

**Comment by nabeelio** [comment]
Quite strange, nothing in the Laravel logs, either? Those shouldn't depend on the domain

**Comment by minnsch** [comment]
Yes, that's why I was confused. I was pretty much able to solve my issues when doing something off the log, well at least a hint. But this issue shows nothing on the log. Pretty weird and pretty frustrated been looking for a solution without having to reset all of my stuff again 😞

**Comment by FatihKoz** [comment]
Any server logs ? Php itself and/or http server (apache/nginx/caddy etc.) Because if something is not in laravel logs it can be failing before it.

**Comment by minnsch** [comment]
Hey Dispo! May I ask where can I find those logs files. I'm using TFDi for my hosting and I have cPanel to go through.

**Comment by minnsch** [comment]
I found these error logs after we made the transfer.


```
[Wed Jan 01 21:54:59.031759 2025] [autoindex:error] [pid 471:tid 471] [client 93.185.162.26:60322] AH01276: Cannot serve directory /home/anavirt3/public_html/assets/fonts/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive
[Wed Jan 01 21:54:03.230758 2025] [autoindex:error] [pid 31699:tid 31699] [client 93.185.162.26:55416] AH01276: Cannot serve directory /home/anavirt3/public_html/assets/img/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive
[Wed Jan 01 21:53:17.910752 2025] [autoindex:error] [pid 31736:tid 31736] [client 93.185.162.26:56932] AH01276: Cannot serve directory /home/anavirt3/public_html/assets/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive
[Wed Jan 01 21:53:16.151561 2025] [autoindex:error] [pid 25089:tid 25089] [client 93.185.162.26:56896] AH01276: Cannot serve directory /home/anavirt3/public_html/uploads/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive
[Wed Jan 01 21:52:53.416088 2025] [autoindex:error] [pid 29986:tid 29986] [client 93.185.162.26:36790] AH01276: Cannot serve directory /home/anavirt3/public_html/.well-known/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive
[Wed Jan 01 21:52:52.160857 2025] [autoindex:error] [pid 27825:tid 27825] [client 93.185.162.26:34786] AH01276: Cannot serve directory /home/anavirt3/public_html/.well-known/pki-validation/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive
AH00112: Warning: DocumentRoot [/home/anavirt3/crew.anavirtualgroup.com/public] does not exist
AH00112: Warning: DocumentRoot [/home/anavirt3/crew.anavirtualgroup.com/public] does not exist
[Tue Dec 31 04:57:12.572930 2024] [autoindex:error] [pid 9440:tid 9440] [client 45.55.213.95:58631] AH01276: Cannot serve directory /home/anavirt3/public_html/uploads/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive, referer: binance.com
[Tue Dec 31 04:57:05.813107 2024] [autoindex:error] [pid 27880:tid 27880] [client 45.55.213.95:59547] AH01276: Cannot serve directory /home/anavirt3/public_html/.well-known/: No matching DirectoryIndex (index.php,index.php7,index.php5,index.perl,index.pl,index.plx,index.ppl,index.cgi,index.jsp,index.jp,index.phtml,index.shtml,index.xhtml,index.html,index.htm,index.js) found, and server-generated directory index forbidden by Options directive, referer: binance.com
```

**Comment by FatihKoz** [comment]
Looks like you are either missing .htaccess files or your server configuration is kind of different.

Not sure though, be sure you have .htaccess enabled and files are present. If you just copy and paste visible files while moving from old domain to new, it is possible to forget hidden/system files.

**Comment by minnsch** [comment]
[anavirtualgroup.com-Jan-2025.gz](https://github.com/user-attachments/files/18292213/anavirtualgroup.com-Jan-2025.gz)
[php.error.log](https://github.com/user-attachments/files/18292215/php.error.log)

I only have a php error log from August nothing very recent from /logs

**Comment by FatihKoz** [comment]
No laravel errors, no php errors, this is slowly going to a point where apache/nginx (http) server not doing its job.

Looks like your request is not being handled at all. If it was the case either php or laravel would have something logged :(

**Comment by minnsch** [comment]
I brought back the .htaccess, tried again assigning a rank, awards, and news after cleaning the cache.... No luck ^^
